java/test/src/main/java/io/ray/test/NodeLabelSchedulingTest.java

```diff
@@ -0,0 +1,35 @@
+package io.ray.test;
+
+import io.ray.api.Ray;
+import io.ray.api.runtimecontext.NodeInfo;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test(groups = "cluster")
+public class NodeLabelSchedulingTest {
+  public void testEmptyNodeLabels() {
+    Ray.init();
+    List<NodeInfo> nodeInfos = Ray.getRuntimeContext().getAllNodeInfo();
+    Assert.assertTrue(nodeInfos.size() == 1);
+    Map<String, String> labels = new HashMap<>();
+    labels.put("ray.io/node_id", nodeInfos.get(0).nodeId.toString());
+    Assert.assertEquals(nodeInfos.get(0).labels, labels);
+    Ray.shutdown();
```

Comment:
We can use "try catch" in tests to improve the robustness, referencing to java/test/src/main/java/io/ray/test/RuntimeEnvTest.java.